src/pnpq/devices/odl_thorlabs_kbd101.py

```diff
@@ -40,6 +40,14 @@ def __init__(
         self,
         serial_port: str | None = None,
         serial_number: str | None = None,
+        name="Thorlabs",
+        model="KBD101 driver DDS100/M Stage",
+        min_move: int | None = None,
```

Comment:
if we have many fields as constructor arguments, we should consider creating a config class.